indico/modules/search/client/js/components/SearchPagination.jsx

```diff
@@ -8,10 +8,13 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import {Pagination} from 'semantic-ui-react';
+
 import './SearchPagination.module.scss';
+import {useResponsive} from 'indico/react/util';
```

Comment:
let's keep css imports at the very end (eslint cannot enforce this because it doesn't know that css imports have no significant side effects :/)

Reply:
Yes, that one slipped through